sdks/python/apache_beam/io/gcp/bigquery_io_read_pipeline.py

```diff
@@ -0,0 +1,93 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+
+# Copyright 2016 Google Inc. All Rights Reserved.
```

Comment:
Did you mean to remove this copyright notice and extra license notice?

Reply:
It's been removed from the code, I think

Reply:
Still on master: https://github.com/apache/beam/blob/b9af7153e6f02bc184b1066c4b0020dc555a2f72/sdks/python/apache_beam/io/gcp/bigquery_io_read_pipeline.py#L19

Reply:
Removed the extra licenses in https://github.com/apache/beam/pull/6627